### ISSUE TITLE
[V3] TopBar Icon color is always blue on iOS 13

### ISSUE THREAD

**Issue by sallar** [wix/react-native-navigation#5651]
### Issue Description

I'm trying to set topBar icons with custom colors, but they're always defaulting to blue on iOS 13.

### Steps to Reproduce / Code Snippets / Screenshots

![image](https://user-images.githubusercontent.com/768052/68719143-f40d0400-05b3-11ea-9699-5dc7cd28fcdb.png)

Same thing happens in playground:

![image](https://user-images.githubusercontent.com/768052/68719159-fcfdd580-05b3-11ea-8726-57437ee573f3.png)

in both instances the icons are supposed to be red. If I remove the color from config, the image color will be used (black).

```js
    rightButtons: [
      {
        id: 'add-shelf',
        icon: require('../../../assets/ui/book-library.png'),
        color: 'red',
      },
```

---
### Environment
* React Native Navigation version: `3.5.1`
* React Native version: `0.61.4`
* Platform(s) (iOS, Android, or both?): `iOS`
* Device info (Simulator/Device? OS version? Debug/Release?): `13.2 Simulator`

**Comment by sallar** [comment]
It seems like this is the same issue as: #5647 I tried their solution (calling `mergeOptions` after mount, and it fixes the issue.

**Comment by yogevbd** [comment]
Fixed in 9f43bca3573e8833eb8219c6e7a829d8d92faac0
